### ISSUE TITLE
window is not defined error only when I have imported the lib and not used it

### ISSUE THREAD

**Issue by jooj123** [elastic/apm-agent-rum-js#644]
I get 

```
ReferenceError: window is not defined
    at eval (webpack-internal:///./node_modules/@elastic/apm-rum-core/dist/es/common/utils.js:43:12)
    at Module../node_modules/@elastic/apm-rum-core/dist/es/common/utils.js (/mypath/build/server/main.js:7920:1)
    at __webpack_require__ (/mypath/build/server/main.js:21:30)
    at eval (webpack-internal:///./node_modules/@elastic/apm-rum-core/dist/es/error-logging/error-logging.js:3:71)
    at Module../node_modules/@elastic/apm-rum-core/dist/es/error-logging/error-logging.js (/mypath/build/server/main.js:7944:1)
    at __webpack_require__ (/mypath/build/server/main.js:21:30)
    at eval (webpack-internal:///./node_modules/@elastic/apm-rum-core/dist/es/error-logging/index.js:2:72)
    at Module../node_modules/@elastic/apm-rum-core/dist/es/error-logging/index.js (/mypath/build/server/main.js:7956:1)
    at __webpack_require__ (/mypath/build/server/main.js:21:30)
    at eval (webpack-internal:///./node_modules/@elastic/apm-rum-core/dist/es/index.js:3:72)
    at Module../node_modules/@elastic/apm-rum-core/dist/es/index.js (/mypath/build/server/main.js:7980:1)
    at __webpack_require__ (/mypath/build/server/main.js:21:30)
    at eval (webpack-internal:///./node_modules/@elastic/apm-rum/dist/es/bootstrap.js:3:79)
    at Module../node_modules/@elastic/apm-rum/dist/es/bootstrap.js (/mypath/build/server/main.js:8159:1)
    at __webpack_require__ (/mypath/build/server/main.js:21:30)
    at eval (webpack-internal:///./node_modules/@elastic/apm-rum/dist/es/index.js:5:68)
```

When i have imported like this: 
`import { init } from '@elastic/apm-rum';`

But I am only calling `init` on componentDidMount (on client)

We have a demo environment which supports server side rendering, so what im trying to do is just execute the rum init on client. 

Ideally `window` should not be called at all if im just doing an import

**Comment by jooj123** [comment]
Added PR: https://github.com/elastic/apm-agent-rum-js/pull/646
Which should fix the issue.
It was because window.performance was being accessed on `import` even though the elastic lib wasnt executed on server render

**Comment by vigneshshanmugam** [comment]
Fixed and released in 4.9.0
